### PR TITLE
Feature/5904 update filter UI

### PR DESCRIPTION
**Pull request by NekoLyn**
update tooltips as per ticket

### CODE REVIEW

File: src/components/common/charts/TimeRangeBarChart.tsx [hunk 1]
```diff
@@ -223,17 +225,32 @@ const TimeRangeBarChart: React.FC<TimeRangeBarChartProps> = ({
...
           valueFormatter: xAxisLabelFormatter,
-          tickMinStep: 3600 * 1000 * 48, // min step: 48h,
-          // If you want the label rotate
-          // tickLabelStyle: {
-          //  angle: 45,
-          //  dominantBaseline: 'hanging',
-          //  textAnchor: 'start',
-          // },
+          tickMinStep: 3600 * 1000 * 48, // min step: 48h
+          label: "Year", // x-axis label
```
Comment (HavierD) [inline src/components/common/charts/TimeRangeBarChart.tsx:230]: if the date range become small, the x-axis will become to "month", and if even smaller, the x-axis will be "day"

File: src/components/common/filters/FilterSection.tsx [hunk 2]
```diff
@@ -1,18 +1,34 @@
...
 import { gap, padding } from "../../../styles/constants";
 
 interface FilterSectionContainerProps {
   title: string;
   children: ReactNode;
   isTitleOnlyHeader?: boolean;
+  toolTip?: string;
 }
 
+const renderTittle = (title: string) => (
```
Comment (HavierD) [inline src/components/common/filters/FilterSection.tsx:13]: no need to extract it. title of tooltip can be set to `undefined`

**Review by HavierD — CHANGES_REQUESTED** [review]

**Comment by HavierD** [comment]
because of the iteration review meeting, i will takeover this branch

**Review by utas-raymondng — APPROVED** [review]
LGTM

**Pull request merged**